systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -292,4 +296,28 @@ void testCruiseControlIntraBrokerBalancing(ExtensionContext extensionContext) {
         KafkaRebalanceStatus kafkaRebalanceStatus = KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(testStorage.getNamespaceName()).withName(testStorage.getClusterName()).get().getStatus();
         assertThat(kafkaRebalanceStatus.getOptimizationResult().get("provisionStatus").toString(), containsString("UNDECIDED"));
     }
+
+    @ParallelNamespaceTest
+    void testCruiseControlIntraBrokerBalancingWithoutSpecifyingJBODStorage(ExtensionContext extensionContext) {
+        final TestStorage testStorage = new TestStorage(extensionContext);
+
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaWithCruiseControl(testStorage.getClusterName(), 3, 3).build());
+        resourceManager.createResource(extensionContext, KafkaRebalanceTemplates.kafkaRebalance(testStorage.getClusterName())
+            .editOrNewSpec()
+            .withRebalanceDisk(true)
+            .endSpec()
+            .build());
+
+        final String coPodName = KubeClusterResource.kubeClient(INFRA_NAMESPACE).getClusterOperatorPodName();
+        final String coLog = StUtils.getLogFromPodByTime(INFRA_NAMESPACE, coPodName, STRIMZI_DEPLOYMENT_NAME, "60s");
+
+        // CO logs must contain warning message
+        assertThat(coLog, CoreMatchers.containsString("Intra-broker balancing only applies to Kafka deployments that use JBOD storage with multiple disks."));
+
+        KafkaRebalanceUtils.waitForKafkaRebalanceCustomResourceState(testStorage.getNamespaceName(), testStorage.getClusterName(), KafkaRebalanceState.ProposalReady);
+        // The provision status should be "RIGHT_SIZED", because we use Ephemeral storage and there is no additional
+        // disks to balance between and also 0 partition movements since there are no disks to balance between.
```

Comment:
Strimzi will force an **inter**-broker balance whenever a JBOD configuration is not found. 
The provision status should be "RIGHT_SIZED" because the cluster should not be under or over provisioned by this **inter**-broker balance. 
When preforming an **intra**-broker balance the provision status should be "UNDECIDED" because moving partitions between disks of the same broker has no effect on whether the cluster is under or over provisioned.

Reply:
Thanks. Comment applied )